pages.pt-BR/common/axel.md

```diff
@@ -0,0 +1,25 @@
+# axel
+
+> Acelerador de downloads.
+> Suporta HTTP, HTTPS, e FTP.
+> Página Oficial: <https://github.com/axel-download-accelerator/axel>.
```

Comment:
Have we been title-casing the URL label? I'd say only the first word should be capitalized, but if that's a pattern already, let's change that across the board in a new PR.

Reply:
Ops. My mistake.